_posts/elements/2015-04-15-icons.md

```diff
@@ -0,0 +1,738 @@
+---
+layout:       element
+title:        Icons
+date:         2015-04-15 00:00:00
+
+categories:   elements
+tags:
+- atomic
+- pattern
+- icons
+
+slug:         icons
+scssurl:      ""
+
+description:  ...
+
+info: ...
+---
+
+<h3 class="hd-6">Media</h3>
+<div class="example-set">
+    <div class="swatch">
+        <div class="icon-display">
+            <svg class="icon icon-video" title="User account"><use xlink:href="{{ site.baseurl }}public/pl/images/edx-svg/svgdefs.svg#icon-video"></use></svg>
```

Comment:
I don't think we want these assets to be in the ``pl/`` directory (meant for all things that render the doc/demo site). These should all be moved over to be assets the base library can use (``/src/static/images``), right? They should be moved to the ``/public`` directory locally by the Gulp workflow.

Please move these.